src/api.ts

```diff
@@ -78,6 +80,12 @@ export const resolvers = {
     settings: (parent: any, args: { settings: Settings }): Settings => {
       return setSettings(args.settings);
     },
+    userEmail: (parent: any, args: { userEmail: string }): boolean => {
+      if (!store.get(USER_EMAIL_KEY, null)) {
```

Comment:
Why not override it if it exists?

Reply:
why yes? why to spend resources on it?

Reply:
Why spend resources checking? It's the same. Also if someone changes users we might want to change the email